src/components/common-components/LargeTextBlock/LargeTextBlock.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
+import './LargeTextBlock.css';
+import Fade from 'react-reveal/Fade';
+
+/**
+ * @returns {React.Component}
+ */
+const LargeTextBlock = props => {
+
+    const [expanded, toggleExpand] = useState(false);
```

Comment:
nit, use `set` for react state stuff, even if it's a boolean.

```suggestion
    const [expanded, setExpanded] = useState(false);
```

Reply:
nit, this file is using spaces instead of tabs. Tabs should be used instead because a developer can define how large they want the tabs to appear in their code editor.